.gitignore

```diff
@@ -46,3 +46,4 @@ styleguide
 # Ignore OSX system files
 .DS_Store
 
+lib/
```

Comment:
so the purpose of the yarn lockfile is to ensure that everyone gets exactly the same version of all dependencies, including transitive dependencies. so like a Gemfile.lock, we need to commit it to version control in order to accomplish that